APIManager.js

```diff
@@ -0,0 +1,83 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {user: {picture: '',
```

Comment:
you do not need to initialize these keys inside the this.data object
You could just have `this.data = {}` and it would still work